spec/javascript/components/__snapshots__/IndexTable.test.js.snap

```diff
@@ -1,5 +1,1669 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`IndexIndexTable Should render Empty string if values are undefined 1`] = `
```

Comment:
@callaghanc we've been talking a bit about how to do snapshots better. Here's a good candidate: this snapshot is ~1600 lines long and very hard to read. Maybe we can pair on finding a better solution?

Reply:
@akegan let's see if we can use the `enzyme-to-json` package that we already have installed, and also if we can think through what needs to be in this snapshot - can we find time today to pair?